docs/core/porting/third-party-deps.md

```diff
@@ -63,31 +61,53 @@ dotnet5.4
 dotnet5.5
 ```
 
-**While these will likely work with your code, there is no guarantee of compatibility**.  Packages with these TFMs were built with pre-release .NET Core packages.  Take note of when (or if) packages like this are updated to be `netstandard`-based.
+**While these TFMs likely work with your code, there is no guarantee of compatibility**. Packages with these TFMs were built with pre-release .NET Core packages. Take note of when (or if) packages using these TFMs are updated to be .NET Standard-based.
 
 > [!NOTE]
-> To use a package targeting a traditional PCL or pre-release .NET Core target, you must use the `imports` directive in your `project.json` file.
+> To use a package targeting a traditional PCL or pre-release .NET Core target, you must use the `PackageTargetFallback` MSBuild element in your project file.
+
+### .NET Framework compatibility mode
+
+After analyzing the NuGet packages, you might find that they only target the .NET Framework, as most NuGet packages do.
+
+Starting with .NET Standard 2.0, the .NET Framework compatibility mode was introduced. This compatibility mode allows .NET Standard and .NET Core projects to reference .NET Framework libraries. Referencing .NET Framework libraries doesn't work for all projects, such as if the library uses Windows Presentation Foundation (WPF) APIs, but it does unblock many porting scenarios.
+
+When you reference NuGet packages that target the .NET Framework on your project, such as [Huitian.PowerCollections](https://www.nuget.org/packages/Huitian.PowerCollections), you get a package fallback warning ([NU1701](/nuget/reference/errors-and-warnings#nu1701)) similar to the following:
+
+`NU1701: Package ‘Huitian.PowerCollections 1.0.0’ was restored using ‘.NETFramework,Version=v4.6.1’ instead of the project target framework ‘.NETStandard,Version=v2.0’. This package may not be fully compatible with your project.`
+
+That warning is displayed when you add the package and every time you build to make sure you test that package with your project. If your project is working as expected, you can suppress that warning by editing the package properties on Visual Studio or by manually editing the project file on your favorite code editor.
```

Comment:
I think "on Visual Studio" and "on your favorite code editor" should use "in" instead of "on".

Reply:
I think you're right. Prepositions are my Achilles heel. @rpetrusha to confirm. 😄